webhyde.py

```diff
@@ -30,9 +30,9 @@ def pages(self):
   def pages(self, pages):
     for p in pages:
       filename = p.lower().replace(" ", "-") + ".html"
-      self.site_yaml["context"]["data"]["menu"].append({"title": p, "url": filename})
-      shutil.copy(self.shortname + "/content/blank.html", self.shortname + "/content/" + filename)
-      replace(self.shortname + "/content/" + filename, "Page name", p)
+      self.site_yaml["context"]["data"]["menu"].append({"title": p, "url": p})
+      shutil.copy(self.shortname + "/content/blank.html", self.shortname + "/content/" + p + ".md")
+      replace(self.shortname + "/content/" + p + ".md", "Page name", p)
```

Comment:
Feel free to take a look at the original version. Filename is only mentioned right there ever. How it worked in Python3 is complete magic to me atm.